src/main/java/net/openhft/chronicle/bytes/BytesUtil.java

```diff
@@ -151,6 +152,29 @@ public static void writeFile(String file, Bytes<byte[]> bytes)
         }
     }
 
+    /**
+     * Returns if the provided nullable BytesStore arguments are equal to each other.
+     *
+     * Consequently, if both arguments are {@code null}, {@code true}
+     * is returned and if exactly one argument is {@code null}, {@code
+     * false} is returned. Otherwise, equality is determined by comparing the content
+     * of the two provided arguments.
+     * <p>
+     * Bytes::equals will eventually be the same as Object::equals and so this
+     * method can provide a bridge between the old behaviour and the new behaviour.
+     *
+     * @param a a ByteStore
+     * @param b a ByteStore to be compared with {@code a} for equality
+     * @return if the arguments are equal to each other
+     * @throws ClosedIllegalStateException if either of the provided parameters are released
+     *
+     * @see Object#equals(Object)
+     */
+    public static boolean contentEqual(@Nullable final BytesStore a,
+                                       @Nullable final BytesStore b) {
+        return BytesInternal.contentEqual(a, b);
```

Comment:
does this throw IllegalStateException ?  javadoc refers to ClosedIllegalStateException

Reply:
It throws `ClosedIllegalStateException` as specified in the JavaDocs. 

```
    public static void throwExceptionIfReleased(final ReferenceCounted referenceCounted) {
        if (referenceCounted.refCount() <= 0) {
            throw new ClosedIllegalStateException("Released");
        }
    }
```